docs/guides/models.md

```diff
@@ -0,0 +1,100 @@
+# Models
+
+In XState, you can model a machine's `context` and `events` externally by using `createModel(...)`. This provides a convenient way to strongly type `context` and `events`, as well as helpers for event creation, assignment and other implementation details in the future.
+
+## `createModel(...)`
+
+The `createModel(...)` function takes
+
+| Argument              | Type   | Description                                                |
+| --------------------- | ------ | ---------------------------------------------------------- |
+| `initialContext`      | object | The initial `context` value                                |
+| `creators` (optional) | object | An object containing various creators (currently `events`) |
```

Comment:
```suggestion
| `creators` (optional) | object | An object containing various creators (`events`) |
```
Makes it clearer without "currently" imo. Or perhaps (currently: `events`. coming soon:  [some known things on the road map])